azure-pipelines-dev.yml

```diff
@@ -0,0 +1,28 @@
+variables:
+- template: /eng/templates/azure-pipelines-shared-variables.yml
+
+trigger:
```

Comment:
Do we want to run both `dev` and `official` pipelines for `main` and `prod`? Wouldn't just `official` be enough for production branches?